untitled1/src/main/com/aca/homework/week10/object/factory/ServletRequestObjectFactory.java

```diff
@@ -0,0 +1,9 @@
+package com.aca.homework.week10.object.factory;
+
+public class ServletRequestObjectFactory<T extends ServletRequest> implements ObjectFactory<ServletRequest> {
```

Comment:
ServletRequestObjectFactory is already an implementation of ObjectFactory<ServletRequest>. so there is no need to define T type parameter here